### PR TITLE
Update community supported README: legal disclaimer

### PR DESCRIPTION
**Pull request by cwarnermm**
Added legal disclaimer to README, as requested by Product.

### CODE REVIEW

**Review by hanzei — CHANGES_REQUESTED** [review]
The repo is not public. This is not needed.

**Comment by mickmister** [comment]
@hanzei This repo is public

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# :warning: Disclaimer
+
+**This repository is community supported and not maintained by Mattermost. Mattermost disclaims any and all liability for integrations, including Third Party Integrations and Mattermost Integrations. All integrations are provided "AS IS", and may be used at your own risk. Integrations may be modified or discontinued at any time.**
```
Comment (cwarnermm) [inline README.md:3]:
```suggestion
# Disclaimer

**This repository is community supported and not maintained by Mattermost. Mattermost disclaims liability for integrations, including Third Party Integrations and Mattermost Integrations. Integrations may be modified or discontinued at any time.**
```

**Comment by cwarnermm** [comment]
@hanzei - Legal's updated guidance and language for the legal disclaimer for community-supported integrations has been applied to this PR.

**Review by mickmister — APPROVED** [review]

**Pull request merged**